### PR TITLE
fix: Fix to the parser handling of `not`.

### PR DESCRIPTION
**Pull request by kouylekov-usit**
The parser handled incorrectly the not operation by finding it in the string variables.

### CODE REVIEW

File: pysquril/tests.py [hunk 1]
```diff
@@ -238,6 +238,10 @@ def run_delete_query(
         assert len(out) == 4
         out = run_select_query('select=d&where=d=not.like.*g3')
         assert len(out) == 2
+        # Run the query with not as a string value.
+        out=run_select_query('select=d&where=d=eq.not')
+        assert len(out) == 0
+        run_select_query('select=d&where=d=not.like.*g3')
```
Comment (leondutoit) [inline pysquril/tests.py:244]: remember to test the output of the query

Comment (kouylekov-usit) [reply pysquril/tests.py:244]: I thought it says empty?

Comment (kouylekov-usit) [reply pysquril/tests.py:244]: oups :(

**Review by leondutoit — APPROVED** [review]
ty!

**Pull request merged**